src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -231,36 +248,39 @@ public Path getJarPath(String jarName) {
   }
 
   /**
-   * Checks whether the Cloud SDK Path with necessary components is installed in
-   * the expected location.
+   * Checks whether the Cloud SDK Path with is valid.
    *
-   * @throws AppEngineJavaComponentsNotInstalledException when the App Engine Java 
-   *     components are not installed in the Cloud SDK
-   * @throws AppEngineException when the Cloud SDK is not installed where expected
+   * @throws CloudSdkNotFoundException when the Cloud SDK is not installed where expected
    */
-  public void validate() throws AppEngineException {
+  public void validateCloudSdk() throws CloudSdkNotFoundException {
     if (sdkPath == null) {
```

Comment:
I think for now we just want to let clients know if the Cloud SDK is valid or not. Details are in the exception message.